cvm/quantization/dataset.py

```diff
@@ -16,7 +16,7 @@
 import logging
 
 dataset_dir = path.expanduser("~/.cvm")
-src = "http://192.168.50.210:8827"
+src = "http://0.0.0.0:8827"
```

Comment:
`src` is set as 0.0.0.0, which seems like not a valid URL. Does it really work?

Reply:
The original src was found invalid when I was tring to dowload the mnist dataset. After revising the address, the dateset could be successfully downloaed.

Reply:
0.0.0.0? It's strange, try to set 127.0.0.1? Maybe the 0.0.0.0 indicates localhost?